### PR TITLE
Add Document for AM/PM `A` time format

### PR DESCRIPTION
**Pull request by firesh**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.
Add Document for AM/PM `A` time format
#8270

### CODE REVIEW

File: examples/docs/en-US/time-picker.md [hunk 1]
```diff
@@ -183,13 +183,13 @@ Can pick an arbitrary time range.
 | end-placeholder | placeholder for the end time in range mode | string | — | — |
 | is-range | whether to pick a time range, only works with `<el-time-picker>` | boolean | — | false |
 | arrow-control | whether to pick time using arrow buttons, only works with `<el-time-picker>` | boolean | — | false |
-| value | value of the picker | Date for Time Picker, and string for Time Select | hour `HH`, minute `mm`, second `ss` | HH:mm:ss |
+| value | value of the picker | Date for Time Picker, and string for Time Select | - | - |
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/time-picker.md]: 这里好像是把可选值误删了？

Comment (firesh) [reply examples/docs/en-US/time-picker.md]: 没有。我看英文文档和中文没有同步，value这块，英文文档应该是之前漏改掉的。

Comment (firesh) [reply examples/docs/en-US/time-picker.md]: image
image
同样的参数，中文文档和英文文档有不同的解释，我认为中文是正确的。

Comment (Leopoldthecoder) [reply examples/docs/en-US/time-picker.md]: 嗯对

**Pull request merged**